### PR TITLE
resolution_pdb_bfactor: Viewer is enhanced

### PR DESCRIPTION
**Pull request by Vilax**

### CODE REVIEW

**Comment by DStrelak** [comment]
Are these changes intended for the release?
Also please follow the naming convention for PR.

**Comment by Vilax** [comment]
> Are these changes intended for the release? Also please follow the naming convention for PR.

As I said in the xmipp repo, it is not mandatory, but if it is possible I will be happy to include them

File: xmipp3/protocols/protocol_resolution_bfactor.py [hunk 1]
```diff
@@ -133,6 +139,7 @@ def convertInputStep(self):
         """
         self.vol = self.mrc_convert(self.localResolutionMap.get().getFileName(),
                                     self._getTmpPath('localResolutionMap.mrc'))
+        print(self.vol)
```
Comment (albertmena) [inline xmipp3/protocols/protocol_resolution_bfactor.py:142]: Is that an intentional print or a debuging print?

File: xmipp3/protocols/protocol_resolution_fso.py [hunk 2]
```diff
@@ -97,6 +97,12 @@ def _defineParams(self, form):
                            ' 3DFSC as anisotropic filter to obtain a directional'
                            'filtered map.')
 
+        form.addParam('filterHalfMaps', BooleanParam, default=True,
+                      condition="estimate3DFSC",
+                      label="Anisotropic filtering of Half Maps",
+                      help='Use the 3DFSCD map as anisotropic filter to directionally'
```
Comment (albertmena) [inline xmipp3/protocols/protocol_resolution_fso.py:103]: add an space after directionally

**Review by albertmena — APPROVED** [review]

**Review by albertmena — APPROVED** [review]

**Pull request merged**